src/PolykeyClient.ts

```diff
@@ -1,10 +1,11 @@
-import type { FileSystem } from './types';
-import type { StreamFactory } from '@matrixai/rpc';
+import type { DeepPartial, FileSystem } from './types';
+import type { WebSocketConfig } from '@matrixai/ws/dist/types';
```

Comment:
Don't import from `dist`, that's only special cases. If the types can't be imported like this then that needs to be fixed on `@matrixai/ws`.

```suggestion
import type { WebSocketConfig } from '@matrixai/ws';
```

Reply:
@amydevs you need to fix this, types currently can't be imported directly

Reply:
Not going to use that type.